src/prefect/telemetry/test_utils.py

```diff
@@ -0,0 +1,105 @@
+from typing import Any, Dict, Protocol, Tuple, Union
```

Comment:
Much of this file was borrowed from opentelemetry-test-utils's TestBase class that several python instrumentation libraries seem to use for testing tho that base class is a unittest base class so I borrowed from there but reflavored it for pytest

Reply:
yeah i can move this module over 👍